shapiq/approximator/marginals/stratified.py

```diff
@@ -0,0 +1,109 @@
+"""This module contains the Stratified Sampling approximation method for the Shapley value (SV) by Maleki et al. (2013).
+It estimates the Shapley values by sampling random marginal contributions grouped by size."""
+
+from typing import Callable, Optional
+
+import numpy as np
+
+from shapiq.approximator._base import Approximator
+from shapiq.interaction_values import InteractionValues
+
+AVAILABLE_INDICES_SHAPIQ = {"SV"}
+
+
+class StratifiedSamplingSV(Approximator):
+    """The Stratified Sampling approximator for estimating the Shapley value (SV).
```

Comment:
Add a quick note what the approx does and point to the reference / paper like in PermutationSamplingSV